### ISSUE TITLE
Postgres: CREATE UNLOGGED unparsable

### ISSUE THREAD

**Issue by PDzikus** [sqlfluff/sqlfluff#1172]
Parsing issue: when parsing code:
```
CREATE UNLOGGED TABLE staging (
    event_type INTEGER
    , event_time TIMESTAMP
    , user_email VARCHAR
    , phone_number VARCHAR
    , processing_date DATE
    , PRIMARY KEY (event_type, event_time, user_email, phone_number, processing_date)
);
```
linter reports unparsable section. The same with parser: unparsable:  !! Expected: 'StatementSegment'

## Expected Behaviour
This is correct code, working as expected - creating an UNLOGGED table. When UNLOGGED keyword is removed, linter works and no errors are detected.

## Observed Behaviour
==== sqlfluff ====
sqlfluff:                0.6.0       python:                  3.8.2
implementation:  cpython  dialect:              postgres
verbosity:            1             templater:               jinja

== [main\models\templates\event_staging_table.sql] FAIL
L:   1 | P:   1 |  PRS | Line 1, Position 1: Found unparsable section: 'CREATE UNLOGGED TABLE staging (\n even...'

## Steps to Reproduce
run linter on the cited code

## Version
sqlfluff, version 0.6.0
Python 3.8.2

## Configuration
```
[sqlfluff]
dialect = postgres
templater = jinja
output_line_length = 120
runaway_limit = 10
#disabled rule L031 [avoid table aliases] - aliasing reduces clutter in code
exclude_rules = L031

[sqlfluff:rules]
tab_space_size = 4
max_line_length = 120
indent_unit = space
comma_style = leading

[sqlfluff:rules:L010]
capitalisation_policy = upper

[sqlfluff:rules:L014]
capitalisation_policy = consistent

[sqlfluff:rules:L030]
capitalisation_policy = upper
```
